GHC/Cmm/Dominators.hs

```diff
@@ -0,0 +1,170 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
we don't have orphan instances here, do we?

Reply:
Good catch!  We don't.